### DISCUSSION TITLE
用PandoraAI整了一个new bing的web客户端

### DISCUSSION THREAD

**Discussion by SoraRoseous** [viopsa233/AI-testers-Forum#27] [技术交流]
地址：[nekointelligence.cyou](https://nekointelligence.cyou/)

不需要科学上网，但是只能用ipv6访问，如果没有的话就用手机移动网络，浏览器夸克或者谷歌都可以。已经有bing账号的小伙伴最好还是把自己的cookie输进去，我怕我号用的人太多然后炸了。默认是没有打开越狱模式的，要自己手动打开才能让预设生效。

**Comment by viopsa233** [comment]
看看

**Comment by Aozaki2501** [comment]
好好好，刚在贴吧看到

**Reply by SoraRoseous** [reply]
我没在贴吧发过诶，是哪个贴啊我想看看

**Reply by Aozaki2501** [reply]
> 我没在贴吧发过那个，是哪个贴啊我想看

https://tieba.baidu.com/p/8354376228?pid=147345590273&cid=0#147345590273 这里的4楼，应该只是同一个框架？

**Reply by viopsa233** [reply]
是的

**Comment by SoraRoseous** [comment]
应该没啥人用吧😂，我调个东西

**Reply by vshshvshshsv19** [reply]
Throttled: Request is throttled.
是什么原因呢？

**Reply by SoraRoseous** [reply]
这是啥呀，能截个图吗，可能是我刚刚重启了吧，你现在再试试呢

**Reply by vshshvshshsv19** [reply]
有大佬说是那个绕过200条限制失效导致的，跟重启没有关系

**Comment by lin-cop** [comment]
![image](https://user-images.githubusercontent.com/131671223/234869873-8ba210be-0bb1-4491-841f-2ee817bf4fa6.png)
不知道出了设么问题

**Reply by SoraRoseous** [reply]
多点几下右下角的刷新就行，我服务的代理可能不太稳定吧，毕竟是很便宜的机场😂

**Comment by SoraRoseous** [comment]
还有用的时候别挂梯子！我的服务器是在国内的，外网反而会访问不到。。

**Reply by MagicEyes123** [reply]
难怪，我人在国外。应该是用不了了

**Reply by viopsa233** [reply]
> 难怪，我人在国外。应该是用不了了

可以试试Cloudflare WARP

**Comment by SoraRoseous** [comment]
我看看能不能搞个ssl，总显示不安全的连接看着好不爽，可能暂时用不了了

**Reply by SoraRoseous** [reply]
加上了，~现在应该安全了~🤣，然而控制台一直在报错，好像是nuxt自带的心跳？~不懂，反正不影响使用就先不管，有空再弄~

**Comment by BiiigOrange** [comment]
大佬你好，我之前在设置里写了自己的预设，但是写完就算开了越狱模式他的回复也会变成普通bing，我想请问一下在哪能找到类似Yuki的预设我学学怎么写qwq

**Reply by SoraRoseous** [reply]
大概就是告诉它你设定的性格，名字，行为方式什么的，具体一点会更好，https://dreamkaka.github.io/ 这个网站上有别人已经弄好的，你可以大概看看，但是不一定非得按照它的来，只要把你自己的设定说清楚就可以了。

**Reply by BiiigOrange** [reply]
好哒好哒，感谢大佬！

**Reply by BiiigOrange** [reply]
dalao好像用不了了qwq我电脑有ipv6但是进不去，手机的火狐浏览器进得去但是AI不回应qwq

**Reply by SoraRoseous** [reply]
好像是我的树莓派炸了，我现在自己都连不上了😂，我现在在别的地方也没办法直接弄它，可能得过几天我回去了才行。好几天没用了，什么时候炸的我都不知道。。。

**Reply by SoraRoseous** [reply]
家里的宽带不知道啥时候又把ip给改了，重启服务后好了。不过长时间用的话还是黑暗哥那个edgeGPT好使，这个也就不方便用电脑的时候应个急，感觉node-chatgpt的问题好多，对话一长bing就变成复读姬了😂

**Reply by BiiigOrange** [reply]
不好意思麻烦dalao了！！这个我用着很舒服！！可能是我目前的需求比较简单，这个对我来说特别的方便qwq谢谢大佬！

**Reply by BiiigOrange** [reply]
今天不知道怎么了一直显示这个 The moderation filter triggered. Try again with different wording.  换个号也还是这个qwq 预设都删了

**Reply by SoraRoseous** [reply]
这好像是node-chatgpt-api的一个bug，我刚刚看了一下最新的issues，然后把我客户端上的代码照着黑暗大佬最新提交的改了一下，现在可以用了。好像黑暗大佬还解决了bing长对话变复读姬的问题？我试了一下24段对话都还是正常的，黑暗大佬牛B！不过每次都开一个新的对话带来的问题就是总是报未登录的错误，不是很清楚什么情况。~无所谓可以用了就不管，多按几次重新生成就是了。而且我才两天没用怎么又寄了😭~

**Reply by BiiigOrange** [reply]
大佬们牛逼！！！！呜呜呜呜完全不懂代码全靠大佬们养活了qwq

**Reply by lover-shadow** [reply]
@BiiigOrange  请问如何联系你，我有些问题想请教你， Q 81892934

**Comment by BiiigOrange** [comment]
大佬大佬，服务器好像寄了qwq网站上不去了

**Reply by BiiigOrange** [reply]
噢没有寄！我本来已经把我家网开了ipv6了，不知道为啥突然用家里的网就上不去了

**Reply by SoraRoseous** [reply]
我这边可以正常用啊，可能是刚开然后路由器还没反应过来？😂

**Comment by BiiigOrange** [comment]
呜呜呜大佬  好像又出问题了  什么unexpected error:200之类的😭😭

**Reply by SoraRoseous** [reply]
用ChatSydney吧😂，更好用。PandoraAI虽然我也可以修但是我懒得弄了。。。感觉没什么必要也没有ChatSydney好用

**Reply by BiiigOrange** [reply]
这样啊  我手机上用的多  我去看看ChatSydney怎么弄！谢谢大佬一直维护！😉😉

**Reply by SoraRoseous** [reply]
更简单，直接用pip装好依赖的库之后用python解释器运行main.py就可以了。手机上用的话建议你用qq浏览器这种x5内核的，而且你可能得在运行main.py的时候改一下代理端口和主机的ip地址，或者直接在代码里改也行，有什么问题再问我吧。
